src/test/java/strategy/AdditionalOperationHandlerTest.java

```diff
@@ -0,0 +1,24 @@
+package strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import db.FruitsDao;
+import db.GenericDao;
+import models.Fruit;
+import org.junit.Test;
+
+public class AdditionalOperationHandlerTest {
+    private static final GenericDao<Fruit, Integer> fruitsDao = new FruitsDao();
+    private static final AdditionalOperationHandler additionalOperationHandler
+            = new AdditionalOperationHandler(fruitsDao);
+    private static final Fruit izir = new Fruit("inzir");
+
+    @Test
+    public void changeBalance_Ok() {
+        additionalOperationHandler.changeBalance(izir.getName(), 1);
+        additionalOperationHandler.changeBalance(izir.getName(), 1);
+        additionalOperationHandler.changeBalance(izir.getName(), 1);
+        int result = fruitsDao.get(izir);
+        assertEquals(3, result);
+    }
+}
```

Comment:
we should clear the storage in the end
let's make it public and address it directly